src/codeu/chat/server/Model.java

```diff
@@ -71,6 +72,28 @@ public int compare(Time a, Time b) {
   public void add(User user) {
     currentUserGeneration = userGenerations.make();
 
+    /// SQLite
+    Connection c = null; /// Create connection
+    Statement stmt = null;
+    try {
+      Class.forName("org.sqlite.JDBC");
+      c = DriverManager.getConnection("jdbc:sqlite:test.db"); /// Connect to database
+      c.setAutoCommit(false); 
+      System.out.println("Opened database successfully model");
+
+      stmt = c.createStatement();
+      ///Test data to be entered into table
```

Comment:
Looks like this is a test of inserting a user into the db. This probably should be removed in the final version, as I don't think you intend to add this test user to ever invocation of the server.